examples/example.krun

```diff
@@ -75,5 +75,5 @@ TEMP_READ_PAUSE = 1
 #PRE_EXECUTION_CMDS = ["sudo service cron stop"]
 #POST_EXECUTION_CMDS = ["sudo service cron start"]
 
-# CPU pinning (off by default)
-#ENABLE_PINNING = False
+# CPU pinning (on by default)
+#ENABLE_PINNING = True
```

Comment:
Actually, maybe it's time to start making some more structured documentation in general. Not sure what format would be best... man, tex, ...?